doc/howtos/building-images.rst

```diff
@@ -12,40 +12,68 @@ Project tools, as explained in the `Yocto Project Quick Start Guide`_.
 Initial Steps
 =============
 
-1. Check out the ``ostro-os`` repository from the ``ostroproject`` GitHub area.  This will retrieve the Ostro OS source code
+#. If your development system is behind a firewall, verify that your proxy settings are configured 
+   to allow access to    the internet for http, https, ftp, and git resources needed by 
+   the Yocto Project build tools, as  explained in `Yocto Project Quick Start: Building Images`_ 
+   and with more details in `Yocto Project: Working Behind a Network Proxy`_
+
+#. Check out the ``ostro-os`` repository from the ``ostroproject`` GitHub area.  ::
+
+   $ git clone https://github.com/ostroproject/ostro-os.git
+
+   This will retrieve the Ostro OS source code
```

Comment:
Cloning this repo will retrieve all recipes that we use in Ostro, downloading the actual source code to be compiled happens when you build the image (or a specific recipe). I'm nit-picking here but if you're so enclined, it would be good to make that subtle distinction in the text.